lua/lspconfig/server_configurations/jdtls.lua

```diff
@@ -77,15 +77,15 @@ local function on_language_status(_, result)
 end
 
 local root_files = {
+  -- Multi-module projects
+  { '.git', 'build.gradle', 'build.gradle.kts' },
```

Comment:
why there add `.git`

Reply:
Sorry for such a late reply.
My current project uses `maven` multi-module.
```
.
├── dependencies
├── generator
├── .git
├── .gitignore
├── .idea
├── modules
├── pom.xml
├── .project
├── README.md
├── .settings
├── common
├── starter
└── upms
```